Tools/ApplicationDebugger/array-transform/README.md

```diff
@@ -236,55 +236,48 @@ please see [Get Started with Intel® Distribution for GDB* on Windows* OS Host](
 ### Example Outputs
 
 ```
-$ gdb-oneapi -q --args ./array-transform cpu
+$ ONEAPI_DEVICE_SELECTOR=*:cpu gdb-oneapi -q ./array-transform
 Reading symbols from ./array-transform...
-(gdb) break 56
-Breakpoint 1 at 0x4057b7: file array-transform.cpp, line 56.
+(gdb) break 54
+Breakpoint 1 at 0x4057b7: file array-transform.cpp, line 54.
 (gdb) run
 ...<snip>...
 [SYCL] Using device: [Intel(R) Core(TM) i9-7900X processor] from [Intel(R) OpenCL]
 [Switching to Thread 0x7fffe3bfe700 (LWP 925)]
 
-Thread 16 "array-transform" hit Breakpoint 1, main::$_1::operator()<cl::sycl::handler>
-(cl::sycl::handler&) const::{lambda(auto:1)#1}::operator()<cl::sycl::item<1, true> >
-(cl::sycl::item<1, true>) const (this=0x7fffe3bfcfa8, index=...) at array-transform.cpp:56
-56              int element = in[index];  // breakpoint-here
+Thread 4 "array-transform" hit Breakpoint 1, main::{lambda(auto:1&)#1}::operator()<sycl::_V1::handler>(sycl::_V1::handler&) const::{lambda(sycl::_V1::id<1>)#1}::operator()(sycl::_V1::id<1>) const (this=0x7fffc85582c8, index=sycl::_V1::id<1> = {...}) at array-transform.cpp:54
+54              int element = in[index];  // breakpoint-here
 (gdb)
 ```
 
 ```
-$ gdb-oneapi -q --args ./array-transform accelerator
+$ ONEAPI_DEVICE_SELECTOR=*:fpga gdb-oneapi -q ./array-transform
 Reading symbols from ./array-transform...
-(gdb) break 56
-Breakpoint 1 at 0x4057b7: file array-transform.cpp, line 56.
+(gdb) break 54
+Breakpoint 1 at 0x4057b7: file array-transform.cpp, line 54.
 (gdb) run
 ...<snip>...
 [SYCL] Using device: [Intel(R) FPGA Emulation Device] from [Intel(R) FPGA Emulation Platform for OpenCL(TM) software]
 [Switching to Thread 0x7fffe1ffb700 (LWP 2387)]
 
-Thread 9 "array-transform" hit Breakpoint 1, main::$_1::operator()<cl::sycl::handler>
-(cl::sycl::handler&) const::{lambda(auto:1)#1}::operator()<cl::sycl::item<1, true> >
-(cl::sycl::item<1, true>) const (this=0x7fffe1ff9fa8, index=...) at array-transform.cpp:56
-56              int element = in[index];  // breakpoint-here
+Thread 6 "array-transform" hit Breakpoint 1, main::{lambda(auto:1&)#1}::operator()<sycl::_V1::handler>(sycl::_V1::handler&) const::{lambda(sycl::_V1::id<1>)#1}::operator()(sycl::_V1::id<1>) const (this=0x7fffc08cef48, index=sycl::_V1::id<1> = {...}) at array-transform.cpp:54
+54              int element = in[index];  // breakpoint-here
 (gdb)
 ```
 
 ```
-$ gdb-oneapi -q --args ./array-transform gpu
+$ ONEAPI_DEVICE_SELECTOR=level_zero:gpu gdb-oneapi -q ./array-transform
 Reading symbols from ./array-transform...
-(gdb) break 56
-Breakpoint 1 at 0x4057b7: file array-transform.cpp, line 56.
+(gdb) break 54
+Breakpoint 1 at 0x4057b7: file array-transform.cpp, line 54.
 (gdb) run
 ...<snip>...
 [SYCL] Using device: [Intel(R) Iris(R) Plus Graphics 650 [0x5927]] from [Intel(R) Level-Zero]
 ...<snip>...
 [Switching to Thread 1073741824 lane 0]
```

Comment:
`Thread 1073741824` remained from the DCD sw stack days, I believe. Could you update it, too?

Reply:
updated.